tennis.py

```diff
@@ -15,8 +15,10 @@ def __init__(self, player1Name, player2Name):
     def won_point(self, playerName):
         if playerName == self.player1.name:
             self.player1.points += 1
-        else:
+        if playerName == self.player2.name:
             self.player2.points += 1
+        else:
+            print(playerName+"is not playing !")
```

Comment:
```suggestion
            print(playerName + " is not playing")
```
* Des espaces autour du `+` pour la lisibilité
* un espace avant le `is` pour que le nom du joueur ne soit pas collé